winml/lib/Api.Image/DeviceHelpers.cpp

```diff
@@ -3,7 +3,9 @@
 
 #include "pch.h"
 
-#include "DirectML.h"
+#if USE_DML
+#include <DirectML.h>
```

Comment:
>DirectML [](start = 10, length = 8)

i think we wanted to have winml always link with DML... and just delay load... right?  that way you can still have a "cpu only" version ... just don't call gpu stuff.